README.md

```diff
@@ -38,8 +40,12 @@ Configure the build from *$TIZENRT_BASEDIR/os/tools* directory
 cd os/tools
 ./configure.sh <board>/<configuration_set>
 ```
-For list of boards and configuration set supported, refer belows.  
-Above copies the canned configuration-set for the particular board, into the *$TIZENRT_BASEDIR/os* directory.  
+The folder list under *build/configs* are \<boards\> and each \<boards\> folder contains \<configuation_set\> folder.  
+Easy way to check the list of \<boards\> and \<configuation_set\> is typing wrong argument like below:
+```bash
+./configue.sh a
+```
+
```

Comment:
./configure.sh --help gets you the list of board and configurations. It is a better choice to present than entering a wrong combination.